src/pages/Dashboard/DashboardViewAdmin.tsx

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable react/display-name */
+import React, { useState, useEffect } from 'react';
+import { format } from 'date-fns';
+import { GridColDef } from '@material-ui/data-grid';
+
+import DataList from 'components/DataList';
+import IconButton from 'components/IconButton';
+import LinkButton from 'components/LinkButton';
+import SubHeader from 'components/SubHeader';
+
+import { UserType, FormattedType } from 'types';
+
+import { Container } from 'styles/dashboard';
+
+interface DashboardViewAdminPros {
+  usersData: UserType[];
+}
+
+const DashboardViewAdmin = ({ usersData }: DashboardViewAdminPros): JSX.Element => {
+  const [formattedData, setFormattedData] = useState<FormattedType[]>([]);
+
+  const columnsUsers: GridColDef[] = [
```

Comment:
Wouldn't it better to move this raw data in a asset or utils folder  and import it into this file? 
So that raw data will be gathered in the same place